Description: Do not run dtls-stress on kFreeBSD. (CLOCK_MONOTONIC)
 kFreeBSD seems to miss CLOCK_MONOTONIC. and therefore cannot run dtls-stress.
Bug-Debian: http://bugs.debian.org/662018
Last-Update: 2012-03-11

diff -NurBbp gnutls-3.0.15/tests/dtls/dtls-stress.c b/tests/dtls/dtls-stress.c
--- gnutls-3.0.15/tests/dtls/dtls-stress.c	2012-02-25 19:55:16.000000000 +0100
+++ b/tests/dtls/dtls-stress.c	2012-03-11 10:01:43.000000000 +0100
@@ -1025,6 +1025,11 @@ int main(int argc, const char* argv[])
 	run_to_end = 1;
 	job_limit = 1;
 
+#if defined __FreeBSD_kernel__ && defined __GLIBC__
+	printf ("Not running dtls-stress due to missing CLOCK_MONOTONIC\n");
+	return 0;
+#endif
+
 #define NEXT_ARG(name) \
 	do { \
 		if (++arg >= argc) { \
